trace/trace_test.go

```diff
@@ -230,6 +230,32 @@ func TestStartSpanWithRemoteParent(t *testing.T) {
 	}
 }
 
+func TestLocalRootSpanID(t *testing.T) {
+	ctx, span1 := StartSpan(context.Background(), "span1")
+	if span1.localRootSpanID == (SpanID{}) {
+		t.Errorf("exporting root span: expected nonzero localRootSpanID")
+	}
+
+	_, span2 := StartSpan(ctx, "span2")
+	if err := checkChild(span1.spanContext, span2); err != nil {
+		t.Error(err)
+	}
+	if got, want := span2.localRootSpanID, span2.localRootSpanID; got != want {
+		t.Errorf("span2.localRootSpanID=%q; want %q (span1.localRootSpanID)", got, want)
+	}
+
+	_, span3 := StartSpanWithRemoteParent(context.Background(), "span3", span2.SpanContext())
+	if err := checkChild(span3.spanContext, span2); err != nil {
+		t.Error(err)
+	}
+	if span3.localRootSpanID == (SpanID{}) {
+		t.Errorf("exporting span with remote parent: expected nonzero localRootSpanID")
+	}
+	if got, want := span3.localRootSpanID, span2.localRootSpanID; got == want {
+		t.Errorf("span2.localRootSpanID=%q; expected different value to span1.localRootSpanID, got same", got)
```

Comment:
Error message should refer to span3 (got) and span2 (different from).

Reply:
Thanks, fixed.